geowebcache/swiftblob/src/main/java/org/geowebcache/swift/TMSKeyBuilder.java

```diff
@@ -116,18 +116,16 @@ public String forTile(TileObject obj) {
         // Key format, comprised of
         // {@code <prefix>/<layer name>/<gridset id>/<format id>/<parameters
         // hash>/<z>/<x>/<y>.<extension>}
-        String key =
-                join(
-                        false,
-                        prefix,
-                        layer,
-                        gridset,
-                        shortFormat,
-                        parametersId,
-                        z,
-                        x,
-                        y + "." + extension);
-        return key;
+        return join(
```

Comment:
Please avoid these changes in the future, they are a simple matter of personal preference, we have no fixed rules to choose one way or the other.